fluent-package/msi/install-test.ps1

```diff
@@ -4,14 +4,34 @@ $msi = ((Get-Item "C:\\fluentd\\fluent-package\\msi\\repositories\\fluent-packag
 Write-Host "Installing ${msi} ..."
 
 Start-Process msiexec -ArgumentList "/i", $msi, "/quiet" -Wait -NoNewWindow
-Start-Service fluentdwinsvc
+$initialStatus = (Get-Service fluentdwinsvc).Status
+if ($initialStatus -ne "Stopped") {
+    Write-Host "The initial status must be 'Stopped', but it was '${initialStatus}'."
+    [Environment]::Exit(1)
+}
 
 $ENV:PATH="C:\\opt\\fluent\\bin;" + $ENV:PATH
 $ENV:PATH="C:\\opt\\fluent;" + $ENV:PATH
 
 td-agent --version
 
-Start-Sleep 3
+# https://github.com/fluent/fluent-package-builder/issues/618
+Write-Host "Measuring times to start the service..."
+$timeSpans = 0..2 | % {
+    Measure-Command { Start-Service fluentdwinsvc }
+    Start-Sleep 15
+    Stop-Service fluentdwinsvc
+    Start-Sleep 15
+}
+Write-Host "Measured seconds to start the service:"
+$timeSpans | %{ Write-Host $_.TotalSeconds }
+if (($timeSpans | Measure-Object -Property TotalSeconds -Maximum).Maximum -gt 6) {
+    # It should be 0.5s ~ 3s because starting service should be done immediately.
+    # (The only things that take time are starting Ruby and loading the libraries.)
+    Write-Host "Launching is abnormally slow."
```

Comment:
It is helpful to show "how slow"  with Write-Host.

e.g.  "Launching is abnormally slow. (>= 6  seconds)"

Reply:
I see! Fixed! Thanks!